darity_test.go

```diff
@@ -34,6 +34,23 @@ func TestAPIVersionKVM(t *testing.T) {
 	}
 }
 
+func TestCreateVM(t *testing.T) {
+	c := &Client{
+		ioctl: func(fd uintptr, request int, argp uintptr) (int, error) {
+			if request != kvmCreateVM {
+				t.Fatalf("unexpected ioctl request number: %d", request)
+			}
+
+			return Version, nil
```

Comment:
I think just returning `1` or something here would be fine.  `Version` doesn't seem quite right.